tests/models.py

```diff
@@ -25,3 +25,5 @@ class MyModel(models.Model):
 
     zero2 = EnumIntegerField(ZeroEnum, default=ZeroEnum.ZERO)
     labeled_enum = EnumField(LabeledEnum, blank=True, null=True)
+
+    zero3 = EnumIntegerField(ZeroEnum, default=0, blank=True)
```

Comment:
What's this field for? I don't see it referenced anywhere else.

Reply:
It's for covering line the last line of `EnumFieldMixin.get_default` but good catch, it should be referenced in some test as well; I'll add it.